example.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/python
-import ms5837
 import time
 
+from MS5837 import ms5837
```

Comment:
Since we are trying to follow pep8, [the package name should be lowercase](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).